python/gtsam/tests/test_NonlinearOptimizer.py

```diff
@@ -83,6 +84,59 @@ def test_graduated_non_convexity(self):
         actual = GncLMOptimizer(self.fg, self.initial_values, gncParams).optimize()
         self.assertAlmostEqual(0, self.fg.error(actual))
 
+    def test_gnc_params(self):
+        base_params = LevenbergMarquardtParams()
+        # Test base params
+        for base_max_iters in (50, 100):
+            base_params.setMaxIterations(base_max_iters)
+            params = GncLMParams(base_params)
+            self.assertEqual(params.baseOptimizerParams.getMaxIterations(), base_max_iters)
+        # Test printing
+        str(params)
```

Comment:
Is this not testing something?

Reply:
It was testing that the params object could actually be printed. This would throw an error if not for the default string I added.

Anyways I made the test a little more explicit now.